src/app/components/card/Card.vue

```diff
@@ -12,8 +12,10 @@
 export interface Props {
   backgroundColor?: string
 }
+
```

Comment:
Let's not add empty lines here, the lines above and are tightly connected. We don't have empty lines here in any existing component.

Reply:
Removed, you can resolve this comment